ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-eus-4.14.yaml

```diff
@@ -134,17 +80,15 @@ tests:
           cpu: 100m
           memory: 200Mi
     - chain: openshift-upgrade-qe-test-eus
-    workflow: cucushift-installer-rehearse-ibmcloud-ipi
-- as: nutanix-ipi-proxy-fips-f28
+    workflow: cucushift-installer-rehearse-ibmcloud-ipi-ovn-ipsec
+- as: nutanix-ipi-proxy-fips-f14
```

Comment:
Nutanix is a small deployment, we can not bear many concurrent running cluster, I suggest to keep the original `f28`

Reply:
I will also update baremetalds-ipi-ovn-ipv4-f14 to f28 in openshift-openshift-tests-private-release-4.16__arm64-nightly-4.16-upgrade-from-eus-4.14.yaml

Reply:
done